index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>Anime Quiz</title>
+	<link rel="stylesheet" href="stylesheet.css"/>
+</head>
+<body>
+    <div id="app">
+		<div class="background-image"></div>
+		<div class="content">
+			<h1>Anime Quiz</h1>
+			<div id="greeting"></div>
+			<button>Go!</button>
+		</div>
+	</div>
+      
+    <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
+    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.25.0/babel.min.js"></script>
+    <script type="text/babel">
+		var name = "Анон"
+		class Hello extends React.Component {
```

Comment:
Please use JS/JSX file for storing this logic. Not html.

Reply:
Moved to separate JSX